processor/src/tasks/processKicadPCB/index.ts

```diff
@@ -1,15 +1,25 @@
 import globule from 'globule'
+import fs from 'node:fs/promises'
 import path from 'node:path'
 import url from 'node:url'
-import { JobData, Job } from '../../job.js'
+import { Job, JobData } from '../../job.js'
 import * as s3 from '../../s3.js'
 import { execEscaped, findKicadPcbFile } from '../../utils.js'
 
 export const outputFiles = ['images/layout.svg'] as const
 
+export interface ProcessKicadPcbData {
+  tmpDir: string
+}
+
 async function processKicadPCB(
   job: Job,
-  { inputDir, kitspaceYaml = {}, outputDir }: Partial<JobData>,
+  {
+    inputDir,
+    kitspaceYaml = {},
+    outputDir,
+    tmpDir,
+  }: ProcessKicadPcbData & Partial<JobData>,
```

Comment:
This type name is confusing

```suggestion
export type tmpDirPath = string
async function processKicadPCB(
  job: Job,
  {
    inputDir,
    kitspaceYaml = {},
    outputDir,
    tmpDir,
  }: tmpDirPath & Partial<JobData>,
```